src/Text/Pandoc/Filter/IncludeCode.hs

```diff
@@ -45,6 +49,7 @@ data InclusionSpec = InclusionSpec
   { include :: FilePath
   , mode    :: InclusionMode
   , dedent  :: Maybe Int
+  , base :: Maybe [Char]
```

Comment:
As a Windows user, I am wary of using strings to represent file-paths. Things might break in unexpected ways. Can you change this to `Maybe FilePath` and propagate the change?